packages/ember-data/lib/system/store.js

```diff
@@ -2135,13 +2108,16 @@ function setupRelationships(store, record, data) {
       relationship = record._relationships.get(key);
       relationship.updateMeta(data.relationships[key].meta);
     }
+
     var value = data.relationships[key].data;
```

Comment:
I can, but not sure I see the point in doing that. If we normalize here, then we will have to check that `value` exists and the `kind` of relationship in two places (here and then again below).